devops/jobs/AppWatcher.groovy

```diff
@@ -81,56 +84,55 @@ class DockerCI {
                     }
                 }
 
-                multiscm {
-                    // check out edx/configuration repository from GitHub
-                    git {
-                        remote {
-                            url(extraVars.get("CONFIGURATION_REPO_URL"))
-                            branch(config_branch)
-                        }
-                        extensions {
-                            relativeTargetDirectory('configuration')
-                            cleanAfterCheckout()
-                            // ignore notifications on commits to branch; this job will be triggered by
-                            // configuration-watcher job
-                            ignoreNotifyCommit()
-                        }
+                // for GitHub Webhooks
+                properties {
+                    if (app_repo) {
+                        githubProjectUrl(extraVars.get("EDX_REPO_ROOT") + app_repo + "/")
```

Comment:
I will add a note about the https/slash restriction above the use of githubProjectUrl, as a heads up to those changing it in the future.